app/models/manageiq/providers/hawkular/datawarehouse_manager.rb

```diff
@@ -38,38 +39,42 @@ def self.verify_ssl_mode
     end
 
     # Hawkular Client
-    def self.raw_connect(hostname, port, token, type = :alerts)
-      klass = case type
+    def self.raw_connect(options)
+      options[:type] ||= :alerts
+      options[:tenant] ||= '_system'
```

Comment:
This mutates the hash passed in.  No immediate bug but IMO better avoided, e.g.
```
type = options[:type] || :alerts
tenant = options[:tenant] || '_system'
```
and then use `type`, `tenant` below.